js/apps/system/_admin/aardvark/APP/react/src/App.js

```diff
@@ -0,0 +1,185 @@
+require('../../frontend/js/arango/arango.js');
+
+// only set this for development
+const env = process.env.NODE_ENV
+if (window.frontendConfig && env === 'development') {
+  window.frontendConfig.basePath = process.env.REACT_APP_ARANGODB_HOST;
+  window.frontendConfig.react = true;
+}
+
+require('../../frontend/js/lib/jquery-ui-1.9.2.custom.min.js');
+require('../../frontend/js/lib/jquery.uploadfile.min.js');
+require('../../frontend/js/lib/bootstrap-min.js');
+
+// Collect all Backbone.js relateds
+require('../../frontend/js/routers/router.js');
+require('../../frontend/js/routers/versionCheck.js');
+require('../../frontend/js/routers/startApp.js');
+
+
+requireAll(require.context(
+  '../../frontend/js/views/'
+));
+
+requireAll(require.context(
+  '../../frontend/js/models/'
+));
+
+requireAll(require.context(
+  '../../frontend/js/collections/'
+));
+
+// Third Party Libraries
+window.tippy = require('tippy.js');
+require('../../frontend/js/lib/bootstrap-pagination.min.js');
+window.numeral = require('../../frontend/js/lib/numeral.min.js'); // TODO 
+window.JSONEditor = jsoneditor;
+// ace 
+window.define = window.ace.define;
+window.aqltemplates = require('../public/assets/aqltemplates.json');
+
+window.d3 = d3;
+require('../../frontend/js/lib/leaflet.js')
+require('../../frontend/js/lib/tile.stamen.js')
+
+// require('../../frontend/js/lib/nv.d3.min.js');
+
+window.prettyBytes = require('../../frontend/js/lib/pretty-bytes.js');
+window.Dygraph = require('../../frontend/js/lib/dygraph-combined.min.js');
+require('../../frontend/js/config/dygraphConfig.js');
+window.moment = require('../../frontend/js/lib/moment.min.js');
+
+// sigma
+window.sigma = Sigma;
+
+// import additional sigma plugins
+import('sigma/build/plugins/sigma.layout.forceAtlas2.min'); // workaround to work with webpack
+
+// additional sigma plugins
+import('../../frontend/js/lib/sigma.canvas.edges.autoCurve.js');
+import('../../frontend/js/lib/sigma.canvas.edges.curve.js');
+import('../../frontend/js/lib/sigma.canvas.edges.dashed.js');
+import('../../frontend/js/lib/sigma.canvas.edges.dotted.js');
+import('../../frontend/js/lib/sigma.canvas.edges.labels.curve.js');
+import('../../frontend/js/lib/sigma.canvas.edges.labels.curvedArrow.js');
+import('../../frontend/js/lib/sigma.canvas.edges.labels.def.js');
+import('../../frontend/js/lib/sigma.canvas.edges.tapered.js');
+import('../../frontend/js/lib/sigma.exporters.image.js');
+import('../../frontend/js/lib/sigma.layout.fruchtermanReingold.js');
+import('../../frontend/js/lib/sigma.layout.noverlap.js');
+import('../../frontend/js/lib/sigma.plugins.animate.js');
+import('../../frontend/js/lib/sigma.plugins.dragNodes.js');
+import('../../frontend/js/lib/sigma.plugins.filter.js');
+import('../../frontend/js/lib/sigma.plugins.fullScreen.js');
+import('../../frontend/js/lib/sigma.plugins.lasso.js');
+import('../../frontend/js/lib/sigma.renderers.halo.js');
+import('../../frontend/js/lib/jquery.csv.min.js');
```

Comment:
Keep in mind unlike `require`, `import` is async, which could theoretically cause race conditions here.

Reply:
Done.